packages/icons/icons.json

```
@@ -891,21 +891,21 @@
     {
         "displayName": "Table",
         "iconName": "th",
-        "tags": "index, rows, columns, agenda, list, spreadsheet",
+        "tags": "index, rows, columns, agenda, list, spreadsheet, table, sheet, entry",
         "group": "table",
         "codepoint": 62186
     },
     {
         "displayName": "Table: list",
         "iconName": "th-list",
-        "tags": "index, rows, list, order, series",
+        "tags": "index, rows, list, order, series, calendar, table",
         "group": "table",
         "codepoint": 62185
     },
     {
         "displayName": "Table: derived",
         "iconName": "th-derived",
-        "tags": "get, obtain, take, acquire, index, rows, columns, list",
+        "tags": "get, obtain, take, acquire, index, rows, columns, list, monocle, table row, row, table, add to dataset, go to dataset",
```

```suggestion
        "tags": "get, obtain, take, acquire, index, rows, columns, list, table, row, add to dataset, go to dataset, provenance",
```

I don't think the monocle product name should leak into blueprint, and it might get renamed down the line anyway